samples/composite-build-logic/settings.gradle.kts

```diff
@@ -0,0 +1,20 @@
+val buildLogicGroup = "composite-build-logic"
+
+listOf("shared", "plugins").forEach { included ->
+    includeBuild("gradle/$included") {
+        dependencySubstitution {
+            substitute(module("$buildLogicGroup:$included")).with(project(":"))
+        }
+    }
+}
+
+// Work around for https://github.com/gradle/gradle-native/issues/522
+pluginManagement {
+    resolutionStrategy {
+        eachPlugin {
+            when (requested.id.id) {
+                "plugins.sample-plugin" -> useModule("$buildLogicGroup:plugins:latest-integration")
+            }
+        }
+    }
+}
```

Comment:
What?! Does this actually work!?

Reply:
Works fine from Gradle perspective. From my testing it seems IDEA can't import it though. I'll get back to this soon.

Reply:
Oh, that's really good information.

Reply:
Oh yeah I do this in my kotlin multiplatform (prior to their recent huge refactor of the MPP plugins) composite projects.  I haven't added native yet, so that might be while I don't have any issues with IDEA imports using this technique.